docs/start_nominating.md

```diff
@@ -96,7 +96,7 @@ To nominate one or more validators, navigate to:
   <img src={useBaseUrl('/nominator-guide/nominate-validator-1.png')} />
 </div>
 
-After clicking on the *Nominate* button you should see a popup called *nominate validators*. Here you can select on or more validators to nominate from the list of available validators. If you select multiple validators, your stake will be automatically assigned to one of your selected validators in a way to maximize decentralization and profits.
+After clicking on the *Nominate* button you should see a popup called *nominate validators*. Here you can select one or more validators to nominate from the list of available validators. It is highly recommended to nominate multiple validators to prevent being inactive if you don't get a spot in one validator (e.g. the validator is overcrowded or not elected into the active validator set). You can nominate up to 16 validators. In every era just one of your nominations can be active, you cannot be chosen by multiple validators simultaneously. Your stake will be automatically assigned to one of your selected validators in a way to maximize decentralization and profits. You just chose the amount of bonded HDX and the validators you trust.
```

Comment:
I would suggest to keep it shorter here and refer to the staking guide for more details (https://github.com/galacticcouncil/HydraDX-docs/pull/26)



It is highly recommended to nominate multiple validators in order to maximize your chances of having your nomination included in the active set of validators (for more information check our staking guide). You can nominate a **maximum of 16 validators**.